spec.bs

```diff
@@ -506,6 +519,209 @@ The <dfn for=Navigator method>runAdAuction(|config|)</dfn> method steps are:
 
 </div>
 
+<div algorithm="construct a pending fenced frame config">
+
+To <dfn>construct a pending fenced frame config</dfn> given an [=auction config=]
+<var ignore>config</var>:
+1. Return a [=fenced frame config=] with the following members:
+  : [=fenced frame config/mapped url=]
+  :: a struct with the following members:
+    : [=mapped url/value=]
+    :: `"about:blank"`
+
+    : [=mapped url/visibility=]
+    :: "<a for=visibility>`opaque`</a>"
+
+  : [=fenced frame config/container size=]
+  :: TODO
+
+  : [=fenced frame config/content size=]
+  :: null
+
+  : [=fenced frame config/interest group descriptor=]
+  :: a struct with the following members:
+    : [=interest group descriptor/value=]
+    :: a struct with the following members:
+      : [=interest group descriptor/owner=]
+      :: ""
+
+      : [=interest group descriptor/name=]
+      :: ""
+
+    : [=interest group descriptor/visibility=]
+    :: "<a for=visibility>`opaque`</a>"
+
+  : [=fenced frame config/on navigate callback=]
+  :: null
+
+  : [=fenced frame config/effective sandbox flags=]
+  :: a struct with the following members:
+    : [=effective sandbox flags/value=]
+    :: TODO
+
+    : [=effective sandbox flags/visibility=]
+    :: "<a for=visibility>`opaque`</a>"
+
+  : [=fenced frame config/effective enabled permissions=]
+  :: a struct with the following members:
+    : [=effective enabled permissions/value=]
+    :: «"<code>[=attribution-reporting=]</code>", "<code>private-aggregation</code>" (TODO:ref),

```

Comment:
```suggestion
    :: «"<code><a spec="attribution-reporting-api">attribution-reporting</a></code>", "<code>private-aggregation</code>" (TODO:ref),
```
to avoid needing the custom anchor, and also file an issue or PR on that spec to export the dfn.

Reply:
Changed and filed an issue on the spec, but removing the custom anchor doesn't seem to work (possibly because the original definition is noexport).